src/main/java/net/sf/jabref/gui/specialfields/SpecialFieldIcon.java

```diff
@@ -0,0 +1,75 @@
+package net.sf.jabref.gui.specialfields;
+
+import javax.swing.Icon;
+import javax.swing.JLabel;
+
+import net.sf.jabref.gui.IconTheme;
+import net.sf.jabref.logic.specialfields.SpecialField;
+import net.sf.jabref.model.entry.SpecialFieldValue;
+
+
+public class SpecialFieldIcon {
+
+    public static Icon getRepresentingIcon(SpecialField field) {
+        if (SpecialField.PRINTED.equals(field)) {
```

Comment:
Hm. When it is already an enum, why not setting the Icon(Name) directly as Enum parameter?
Then we neither need a switch nor a hashmap